pygmt/clib/session.py

```diff
@@ -2068,54 +2061,49 @@ def virtualfile_to_raster(
             }[family]
         return self.read_virtualfile(vfname, kind=kind).contents.to_dataarray()
 
-    def extract_region(self):
+    def extract_region(self) -> np.ndarray:
         """
-        Extract the WESN bounding box of the currently active figure.
+        Extract the region of the currently active figure.
 
-        Retrieves the information from the PostScript file, so it works for
-        country codes as well.
+        Retrieves the information from the PostScript file, so it works for country
+        codes as well.
 
         Returns
         -------
-        * wesn : 1-D array
-            A numpy 1-D array with the west, east, south, and north dimensions
-            of the current figure.
+        region
+            A numpy 1-D array with the west, east, south, and north dimensions of the
+            current figure.
 
         Examples
         --------
-
         >>> import pygmt
         >>> fig = pygmt.Figure()
         >>> fig.coast(
-        ...     region=[0, 10, -20, -10],
-        ...     projection="M6i",
-        ...     frame=True,
-        ...     land="black",
+        ...     region=[0, 10, -20, -10], projection="M12c", frame=True, land="black"
         ... )
```

Comment:
```suggestion
        ...     region=[0, 10, -20, -10], projection="M12c", frame=True, land="black",
        ... )
```

Reply:
I don't think `ruff` likes this trailing comma.

Reply:
OK. I was wondering, if this comma is required. But if not simply ignore this review.